docs/04-operation-guides/troubleshooting/eventing/evnt-05-fix-pending-messages.md

```diff
@@ -68,14 +74,20 @@ You can see, that its leader is the `eventing-nats-1` replica.
 
 #### Find the broken consumers using the NATS cli
```

Comment:
This seems like the much easier method. Why not put it first, before the Grafana way? Especially because line 27 anyway says it's a prerequisite to have NATS cli installed. So why would people prefer the Grafana way?